.cirrus.yml

```diff
@@ -14,7 +14,11 @@ freebsd_task:
     env:
         HOME: /home/testuser
 
-    install_script: pkg install -y rust gmake rubygem-asciidoctor pkgconf stfl curl json-c ncurses openssl111 sqlite3 gettext-tools libxml2
+    install_script:
+        - sed -i.bak -e 's,pkg+http://pkg.FreeBSD.org/\${ABI}/quarterly,pkg+http://pkg.FreeBSD.org/\${ABI}/latest,' /etc/pkg/FreeBSD.conf        
```

Comment:
Is this change really necessary? I understand "latest" might help us detect incompatibilities sooner, but on the other hand, it can result in spurious breakage as something changes in FreeBSD. Also, since "quarterly" is the default, it's very likely that our users run that too. We might miss some problems with older versions of packages because we're already on the newer ones.

Reply:
Since I had to use a 11.3-STABLE snapshot (which defaults to 'latest', while 12.1-RELEASE defaults to 'quarterly'), we'll have to make this change in one direction one way or another (either from latest->quarterly or quarterly->latest). The Newsboat package is only ever updated in 'latest' so far.

Reply:
Got it. Thanks for the explanation! Let's use laltest, then. If it breaks too often, we can always reconsider.

Can you condense this whole conversation into a comment in the Cirrus config, please?

Reply:
Done.